src/unix/process.c

```diff
@@ -326,10 +326,8 @@ static void uv__process_child_init(const uv_process_options_t* options,
 
   /* Reset signal mask. */
   sigemptyset(&set);
-  err = pthread_sigmask(SIG_SETMASK, &set, NULL);
-
-  if (err != 0)
-    uv__write_errno(error_fd);
+  if (sigprocmask(SIG_SETMASK, &set, NULL) != 0)
```

Comment:
Is there a reason for using `sigprocmask()` instead of `pthread_sigmask()`?

Reply:
pthreads isn't usually marked async-safe (though this function is, as this particular function is defined to just be a trivial wrapper around `sigprocmask`) and so may not be accessible after the fork, so sigprocmask seemed formally more correct here